pimd/pim_oil.c

```diff
@@ -563,6 +563,12 @@ int pim_channel_oil_empty(struct channel_oil *c_oil)
 		null_oil.mfcc_ttls[0] = 1;
 #else
 		IF_SET(0, &null_oil.mf6cc_ifset);
+#endif
+	} else {
+#if PIM_IPV == 4
```

Comment:
oil_if_set api can be used here instead for both if and else.

Reply:
fixed